python/lsst/sims/utils/ObservationMetaData.py

```diff
@@ -94,12 +97,22 @@ def __init__(self, boundType=None, boundLength=None,
 
         self._bounds = None
         self._boundType = boundType
-        self._mjd = mjd
         self._bandpass = bandpassName
         self._skyBrightness = skyBrightness
         self._site = site
         self._epoch = epoch
 
+        if mjd is not None:
+            if isinstance(mjd, float):
+                self._mjd = ModifiedJulianDate(TAI=mjd)
```

Comment:
You might consider letting users pass an int here too -- for someone being super casual about their observation metadata, they might just pass in a 'day' (like 49353 or our new start day of 59580).